src/GitHub.Api/Git/GitClient.cs

```diff
@@ -148,9 +152,39 @@ private Task<NPath> LookForPortableGit()
             return path;
         }
 
-        public bool ValidateGitInstall(NPath path)
+        public ITask<ValidateGitInstallResult> ValidateGitInstall(string path)
         {
-            return path.FileExists();
+            if (!path.ToNPath().FileExists())
+            {
+                return new FuncTask<ValidateGitInstallResult>(cancellationToken, 
+                    () => new ValidateGitInstallResult(false, null, null));
+            }
+
+            Version gitVersion = null;
+            Version gitLfsVersion = null;
+
+            var gitVersionTask = new GitVersionTask(cancellationToken);
+            gitVersionTask.Configure(processManager, path, 
+                gitVersionTask.ProcessArguments, environment.RepositoryPath);
+
+            var gitLfsVersionTask = new GitLfsVersionTask(cancellationToken);
+            gitLfsVersionTask.Configure(processManager, path,
+                gitLfsVersionTask.ProcessArguments, environment.RepositoryPath);
+
+            return gitVersionTask
+                .Then((result, version) => { gitVersion = version; })
```

Comment:
Nitpicking the braces and semicolon here, no real need for them. :nitpick: :nitpick:

Reply:
👍